src/api/status/public/css/terminal.css

```diff
@@ -1,3 +1,6 @@
 .xterm {
   padding: 0.5rem;
+  position: relative;
+  width: 100vh;
+  height: 89vh;
```

Comment:
I think `vh` is the correct way, but seem like we need to apply it on a different class or in combination with another class.

In the picture, the text should go all the way to the bottom, but it turns to scroll at half the height.

![image](https://user-images.githubusercontent.com/52666982/144909177-43dc729f-a9b8-4db6-82c3-49a5e286510e.png)

Reply:
I deleted the position and width properties. I know why the text turns to scroll at half the height. I think you have two screen and you changed the screen from one to another. When we change the screen, we need to refresh it.

Reply:
Yeah you're right.